purchase_order_general_discount/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Purchase Order General Discount",
+    "summary": "General discount per purchase order",
+    "version": "14.0.1.0.0",
+    "category": "Purchases",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "author": "Tecnativa, Aures TIC, Odoo Community Association (OCA)",
```

Comment:
This field is intended to show original author of the module, no migrations, neither minor changes contributors.